examples/characterize_chromatographic_system/binding_model_parameters.md

```diff
@@ -264,7 +300,12 @@ if __name__ == '__main__':
         )
 
 
-    optimization_problem.add_callback(callback, requires=[simulator])
+    optimization_problem.add_callback(callback, requires=[simulator], callbacks_dir="tmp")
```

Comment:
is the callbacks_dir required here?

Reply:
Without that, the optimization_problem.evaluate_callbacks(ind) in line 308 throws an error.